CRAYSAT-1626: Document workaround for `sat bootsys` bug (#2841)
Cray-HPE/docs-csm: operations/power_management/Power_On_Compute_Cabinets.md

   ![PSU Status LEDs](../../img/operations/PSU_Status.svg)

> 1. (`ncn-m001#`) Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets chassis and slots.

Reviewer: ```suggestion
1. (`ncn-m001#`) Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets, chassis, and slots.
```

   ```console
   sat bootsys boot --stage cabinet-power
   ```

   This command first resumes the `hms-discovery` Kubernetes cronjob and waits for it to be
   scheduled. Then the `hms-discovery` job initiates power-on of the liquid-cooled cabinets.
   Then the `sat bootsys` command waits for the components in the liquid-cooled cabinets to be
   powered on. This command controls power only to liquid-cooled cabinets.

   The `sat bootsys` command may time out while waiting for the `hms-discovery` cronjob to be
   scheduled with the following message:

   ```text
   ERROR: The cronjob hms-discovery in namespace services was not scheduled within expected window after being resumed.
   ```

>    If this occurs, first check if the cronjob needs to be re-created by following the instructions

Reviewer: ```suggestion
   If this occurs, first check if the cronjob needs to be re-created. To do this, follow the instructions
```

> 1. Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets chassis and slots.

Reviewer: ```suggestion
1. Use the System Admin Toolkit \(`sat`\) to power on liquid-cooled cabinets, chassis, and slots.
```

   ```console
   sat bootsys boot --stage cabinet-power
   ```

>    This command first resumes the `hms-discovery` Kubernetes cronjob and waits for it to be
>    scheduled. Then the `hms-discovery` job initiates power-on of the liquid-cooled cabinets.
>    Then the `sat bootsys` command waits for the components in the liquid-cooled cabinets to be
>    powered on. This command controls power only to liquid-cooled cabinets.

Reviewer: ```suggestion
   This command first resumes the `hms-discovery` Kubernetes cronjob and waits for it to be
   scheduled. Then, the `hms-discovery` job initiates power-on of the liquid-cooled cabinets.
   Finally, the `sat bootsys` command waits for the components in the liquid-cooled cabinets to be
   powered on. The `sat bootsys` command controls power only to liquid-cooled cabinets.
```

   The `sat bootsys` command may time out while waiting for the `hms-discovery` cronjob to be
   scheduled with the following message:

   ```text
   ERROR: The cronjob hms-discovery in namespace services was not scheduled within expected window after being resumed.
   ```

   If this occurs, first check if the cronjob needs to be re-created by following the instructions
   in the [Check `cronjobs`](Power_On_and_Start_the_Management_Kubernetes_Cluster.md#check-cronjobs)
   section of the [Power On and Start the Management Kubernetes Cluster](Power_On_and_Start_the_Management_Kubernetes_Cluster.md)
   procedure.

   If the cronjob does not need to be re-created and has been scheduled within the time expected

>    based on its cron schedule, then execute the `sat bootsys boot --stage cabinet-power` command

Reviewer: ```suggestion
   (based on its cron schedule), execute the `sat bootsys boot --stage cabinet-power` command
```
